contracts/bonding/BondingManager.sol

```diff
@@ -650,8 +653,14 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
     }
 
     /**
-     * @dev Return transcoder information
+     * @notice Return transcoder information
      * @param _transcoder Address of transcoder
+     * @return trancoder's last reward round
+     * @return transcoder's reward cut
+     * @return transcoder's fee share
+     * @return round in which transcoder's stake was last updated
```

Comment:
"round in which trancoder's stake was last updated while active"?

Reply:
fixed in 8d0bb3e